artifacts/flagger/crd.yaml

```
@@ -806,6 +806,9 @@ spec:
                                 - PASSTHROUGH
                                 - LEAST_REQUEST
                               type: string
+                            warmupDurationSecs:
+                              description: Grace time to receive complete request
```

```suggestion
                              description: Represents the warmup duration of Service.
```

same suggestion also applies to the other files

---

@aryan9600 

Thanks for suggesting the correction!
I've made the corrections and combined the commits into one, so I hope you can check it out!

https://github.com/fluxcd/flagger/pull/1540/commits/d196fae71cf009ed8c99dc8653aeb488e7a50e18